emission/analysis/modelling/trip_model/run_model.py

```diff
@@ -97,7 +98,8 @@ def update_trip_model(
 
 
 def predict_labels_with_n(
-    trip: ecwc.Confirmedtrip,
+    user_id: UUID,
```

Comment:
you should still be able to infer the `user_id` from the trip_list. you just have to assert that all the user_ids are the same and then use that unique one.